mysql-test/suite/galera/t/lp1376747-4.test

```diff
@@ -28,7 +28,7 @@ INSERT INTO t1 VALUES (2,3);
 SET session wsrep_sync_wait=0;
 SET session wsrep_causal_reads=OFF;
 SHOW CREATE TABLE t1;
-FLUSH TABLES t1 WITH READ LOCK;
+--send FLUSH TABLES t1 WITH READ LOCK;
```

Comment:
Did you ran this test. I think it will generate a diff and result file will need re-recording. I don't see result for this test ?